library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
+	parameter x,y,w,h.
+	// w and h are INTERNAL width and height
```

Comment:
In new version, I've changed it to be x,y of upper left corner (as it was previously), and external w,h. Turns out the only things that had to be changed were example (fit to screen calculation) and window drawing routine.